invenio_oaiserver/config.py

```diff
@@ -48,6 +48,13 @@
 OAISERVER_METADATA_FORMATS = {
     'oai_dc': {
         'serializer': 'dojson.contrib.to_marc21.utils:dumps',
+        'etree_dumper': (
```

Comment:
Since the output is **always** xml then we can say that the `etree` is the serialized object.